internal/devserver/main/BUILD

```diff
@@ -24,9 +24,39 @@ filegroup(
 
 load("@io_bazel_rules_go//go:def.bzl", "go_binary")
 
+genrule(
+   name = "gen_javascript",
+   outs = ["javascript.go"],
+   srcs = ["@//:node_modules"],
+   cmd = """
+cat > $(location :javascript.go) <<EOF
+// @filedescription DO NOT EDIT. THIS FILE IS AUTOMATICALLY GENERATED.
+package main
+
+const RequireJs string = \\`
+EOF
+
+cat node_modules/requirejs/require.js >> $(location :javascript.go)
```

Comment:
$(location @//:node_modules) doesn't work as its a file group. This is assuming the location is node_modules/* here.

I could something like this instead:

```
NODE_MODULES="$(locations @//:node_modules)"
```

and then grep out the locations for `require.js` and `livereload.js` from the list of strings.

Reply:
Updated to grep locations from $(locations @//:node_modules)